tic-tac-toe-app/index.html

```diff
@@ -4,14 +4,14 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="styles.css">
-  <title>Document</title>
+  <title>Tic-Tac-Toe</title>
 </head>
 <body>
   <main class="container"></main>
   
   <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
   <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-standalone/6.26.0/babel.js"></script>
-  <script type="text/babel" src="app.jsx"></script>
+  <script type="text/babel" src="https://cdnjs.cloudflare.com/ajax/libs/babel-standalone/6.26.0/babel.js"></script>
+  <script src="App.jsx"></script>
```

Comment:
In order to quickly spin up react in a browser, the started code base uses the scripts on lines 12-14. This `type="text/babel"` line needs to be on line 15 like so:

```suggestion
  <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-standalone/6.26.0/babel.js"></script>
  <script type="text/babel" src="App.jsx"></script>
```